x/distribution/testutil/staking_helper.go

```diff
@@ -49,13 +49,19 @@ func SlashValidator(
 	slashFactor math.LegacyDec,
 	validator *stakingtypes.Validator,
 	distrKeeper *keeper.Keeper,
+	sk *MockStakingKeeper,
```

Comment:
Makes sense, too bad we cannot just get the one from the distribution keeper. But i guess that makes sense as it would be ugly otherwise.
We do need a changelog.

Reply:
since staking is private i dont have access to it, otherwise i would have loved to do this